### ISSUE TITLE
a11y: list items for selecting folders, drives and recents have no automation name.

### ISSUE THREAD

**Issue by beqabeqa473** [files-community/Files#5078]
Hello.

i am using the latest version of files app built from the latest codebase.

list items mentioned in title are not readable by the screenreader.

It would be great if you add corresponding names on them.

**Comment by yaichenbaum** [comment]
@chingucoding Do you want to take a look at this one?

**Comment by chingucoding** [comment]
I can take a look at this, not sure when I will get to this though.

**Comment by yaichenbaum** [comment]
@beqabeqa473 This issue should be resolved with #5197.

**Comment by chingucoding** [comment]
Hey @beqabeqa473 I'm not sure what you are referring to with this issue, can you please provide a more detailed explanation or maybe show in a screenshot which elements don't have an appropriate UIA name? :)
